swap_meet/clothing.py

```diff
@@ -0,0 +1,9 @@
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, category="Clothing", condition=0, age=0):
+        super().__init__(category, condition, age)
```

Comment:
👍  this works perfectly! But it still lets the user change the category, and we don't want that. class Clothing will always have the category "clothing." So let's try this:

```suggestion
    def __init__(self, condition=0, age=0):
        super().__init__("clothing", condition, age)
```